src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -23662,7 +23662,148 @@ public void Main()
         }
 
         [Fact]
-        public void AssertsTrue_NotNull()
+        public void DoesNotReturn_Simple()
```

Comment:
Is it worth having a test with a definite assignment error, to demonstrate that this only affects nullable?